webclient/src/components/Registration/RegistrationDialog.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";

```

Comment:
The directory name should match the file name should match the component name.

`/components/RegistrationDialog/RegistrationDialog.tsx`